.github/ISSUE_TEMPLATE/flaking-test.yaml

```diff
@@ -0,0 +1,47 @@
+name: Flaking Test
+description: Report flaky tests or jobs in OVN Kubernetes CI
+labels: kind/ci-flake
+body:
+  - type: textarea
+    id: jobs
+    attributes:
+      label: Which jobs are flaking?
+      description: |
+        Please only use this template for submitting reports about flaky tests or jobs (pass or fail with no underlying change in code) in Kubernetes CI.
+        Links to failed CI runs which show the specific failures in presubmit CI runs are appreciated.
+        Is it shard, control-plane, dualstack, unit-tests etc?
+    validations:
+      required: true
+
+  - type: textarea
+    id: tests
+    attributes:
+      label: Which tests are flaking?
+      description: |
+        Please tell us the name of the tests that are flaking
+    validations:
+      required: true
+
+  - type: textarea
+    id: since
+    attributes:
+      label: Since when has it been flaking?
+    validations:
+      required: true
+
+  - type: input
```

Comment:
should we leave this out until we have a test grid?

Reply:
I think @jluhrsen was ready with a test grid, I think maybe until then we can skip this yea. Let me perhaps comment out this bit with a FIXME instead